### ISSUE TITLE
[Public interface]: Masthead image caption styling (accessibility + legibility)

### ISSUE THREAD

**Issue by aturner** [ucldc/cinco#539]
Revisit masthead image caption (`LeConte Family Papers / UC Berkeley, Bancroft Library` in white text), to see if we can re-style it along the lines of the wireframe for accessibility and legibility -- black text over opaque background: https://www.figma.com/proto/dhNDtXnBhFEZrgmXRcCKpp/OAC5?node-id=8025-55575&t=NxnU73GLGgpi7DlR-0&scaling=contain&content-scaling=responsive&page-id=5351%3A39118&hide-ui=1
